app/javascript/react/components/Button/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import * as S from "./Button.style";
+import React from "react";
```

Comment:
We can agree that we always first import from libraries, leave an empty line, custom components and utils, empty line and styles at the end. :) It's just a matter of following convention. :)

Reply:
sure
@jankrzempek I think that this comment might interest you, too :D